src/main/java/uk/ac/ebi/ampt2d/accession/object/AccessionObject.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.object;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.commons.lang3.StringUtils;
+import uk.ac.ebi.ampt2d.accession.AccessioningObject;
+
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+@Entity
+public class AccessionObject implements AccessioningObject<String> {
```

Comment:
The difference between these 2 types is not clear, and because there is not a huge difference between them, I suggest using:

* `IAccessionedObject` instead of `AccessioningObject`
* `AccessionedObject` instead of `AccessionObject`